### PR TITLE
created uploadPage, fix backend upload

### PR DESCRIPTION
**Pull request by minhphan156**

### CODE REVIEW

**Comment by Hanasou** [comment]
What's the expected behavior? I ran npm start on back-end and front-end. App started on localhost:3000 but clicking on "Upload File" did nothing.

File: src/components/UploadPage.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            onupdatefiles={fileItems => {
+              this.setState({
+                file: fileItems.map(fileItem => fileItem.file)
+              });
+            }}
+          />
+          <Button
+            type="submit"
+            variant="contained"
+            color="default"
```
Comment (evanugarte) [inline src/components/UploadPage.js]: @Hanasou There is no `onClick` function here. so as expected, nothing happens.

File: .gitignore [hunk 2]
```diff
@@ -3,6 +3,7 @@
 # dependencies
 /node_modules
 /back-end/node_modules
+/back-end/config
```
[the remark this answers is not in the document]
Comment (minhphan156) [reply .gitignore]: fixed

File: src/Routing.js [hunk 3]
```diff
@@ -20,6 +23,9 @@ function Routing({ appProps }) {
         <PrivateRoute exact path="/SocialPage"
           appProps={{ allowed: appProps.authenticated, ...appProps }}
           component={SocialPage} />
+        <PrivateRoute exact path="/UploadPage"
```
[the remark this answers is not in the document]
Comment (minhphan156) [reply src/Routing.js]: fixed

**Comment by minhphan156** [comment]
> What's the expected behavior? I ran npm start on back-end and front-end. App started on localhost:3000 but clicking on "Upload File" did nothing.

I fixed it! now you can click upload file to see the page

File: back-end/src/routes/api.js [hunk 4]
```diff
@@ -28,79 +33,86 @@ const uploadFile = (request, response) => {
...
-                "email": request.body.email //Change to request.user.email when authentication is set up
+              "file": newFileName,
+              "tags": labelNames,
+              "desc": request.body.desc,
+              "email": request.body.email //Change to request.user.email when authentication is set up
             }
           };
 
           const fileData = await dynamodb.put(fileParams).promise();
+          // eslint-disable-next-line
```
Comment (evanugarte) [inline back-end/src/routes/api.js]: please get rid of these disable comments. instead, add to the [.eslintrc.json file](https://github.com/evanugarte/SmartPhotoCategorizer/blob/master/.eslintrc.json).

change all occurances of `console.log` to `console.debug` and change [this line](https://github.com/evanugarte/SmartPhotoCategorizer/blob/06ab716b9a476a201c732b616a499d73fd0d6f08/.eslintrc.json#L49) to
`"no-console": ["error", { "allow": ["debug", "error"] }]`

File: src/components/UploadPage.js [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+      description: "",
+      file: null,
+      isSubmit: false
+    };
+    this.onChange = this.onChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+
+    this.handleInit = this.handleInit.bind(this);
+  }
+  //   componentWillMount() {
```
Comment (evanugarte) [inline src/components/UploadPage.js]: this is handled by `Routing.js`, delete?

File: back-end/src/routes/api.js (hunk 4 above)
Comment (minhphan156) [reply back-end/src/routes/api.js]: done

File: back-end/src/routes/api.js [hunk 6]
```diff
@@ -1,13 +1,18 @@
 // Load the SDK for JavaScript
-var AWS = require('aws-sdk');
-var uuid = require('uuid'); // Use this later to generate a unique identifier to prevent collisions?
+var AWS = require("aws-sdk");
+var uuid = require("uuid"); // Use this later to generate a unique identifier to prevent collisions?
 //var fs = require('fs');
```
Comment (evanugarte) [inline back-end/src/routes/api.js]: delete unecesarry comments

**Pull request merged**